### PR TITLE
Gpuciscripts clean and update

### PR DESCRIPTION
**Pull request by msadang**
In CI folder the following changes have been made:

- Remove logger function and replace all logger calls with gpuci_logger

- Removed all ellipses ... from logger messages

- Prepend /opt to the conda path in PATH variable

- Replace conda with gpuci_conda_retry for build and install calls  (Did not replace conda activate with gpuci_conda_retry) 

- Replace source activate <env> with the following:
. /opt/conda/etc/profile.d/conda.sh
conda activate rapids

- Replace conda list with more verbose information:
conda info
conda config --show-sources
conda list --show-channel-urls

- Update Copyright year in the top of scripts to include 2020 if applicable

- Set PARALLEL_LEVEL to ${PARALLEL_LEVEL:-4}

- Set gpuci_conda_retry flags in cpu/build.sh
Setup 'gpuci_conda_retry' for build retries (results in 2 total attempts)
export GPUCI_CONDA_RETRY_MAX=1
export GPUCI_CONDA_RETRY_SLEEP=30

- Replace calls to gcc and g++ with $CC and $CXX respectively

### CODE REVIEW

**Comment by msadang** [comment]
gpuci_logger and parallel level changed.

File: ci/cpu/build.sh [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-# Copyright (c) 2018, NVIDIA CORPORATION.
+# Copyright (c) 2020, NVIDIA CORPORATION.
 set -e
 
 # Logger function for build status output
-function logger() {
+function gpuci_logger() {
   echo -e "\n>>>> $@\n"
 }
```
Comment (dillon-cullinan) [inline ci/cpu/build.sh:8]: Should be removed.

File: ci/cpu/build.sh [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 # Set path and build parallel level
-export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
-export PARALLEL_LEVEL=4
+export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=-4
```
Comment (dillon-cullinan) [inline ci/cpu/build.sh:12]:
```suggestion
export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
```

File: ci/cpu/build.sh [hunk 3]
```diff
@@ -49,16 +56,16 @@ conda config --set ssl_verify False
...
 
-logger "Build conda pkg for frigate..."
-conda build conda/recipes/frigate --python=$PYTHON
+gpuci_logger "Build conda pkg for frigate"
+gpuci_conda_retry build conda/recipes/frigate --python=$PYTHON
 
-logger "Build pip pkg for frigate..."
+gpuci_logger "Build pip pkg for frigate"
 rm -rf dist/
 python setup.py sdist bdist_wheel
```
Comment (dillon-cullinan) [inline ci/cpu/build.sh:64]: This can be removed, we don't build pip packages anymore.

**Review by dillon-cullinan — CHANGES_REQUESTED** [review]

File: ci/cpu/build.sh (hunk 1 above)
Comment (msadang) [reply ci/cpu/build.sh:8]: completed

File: ci/cpu/build.sh (hunk 3 above)
Comment (msadang) [reply ci/cpu/build.sh:64]: Completed

**Review by dillon-cullinan — CHANGES_REQUESTED** [review]
My fault, we actually do build this package for pypi, it should be added back.

**Comment by msadang** [comment]
rerun tests

**Pull request merged**